sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1007,6 +1007,7 @@ private void SetNameFieldsIfNull()
                 rangeGetContentHash.ToValidationOptions(),
                 progressHandler: default,
                 $"{nameof(BlobBaseClient)}.{nameof(Download)}",
+                null,
```

Comment:
Nit - whenever you're passing a literal value like this, it helps a lot to use named parameters like `userPrincipalName: null`.  Same with `async: false` below.  This helps folks reading the code and it makes it easier to refactor things if we end up with new overloads taking more parameters in the future.